### ISSUE TITLE
Implement Qfeed redesign for Mobile and Desktop

### ISSUE THREAD

**Issue by faradayadmin** [faradayafrica/Faraday-frontend#256]
Strictly stick to implementing the design on mobile and desktop. 
That is implement the redesign and keep the ability to vote which has already been implemented. 

Other actions on Qfeed such as Echo, Pen, Sharing are still being designed once the design is ready, a separate issue would be created for them.

**Comment by devgenix** [comment]
@jomefavourite assigning this to you.
